pkg/workceptor/command.go

```diff
@@ -11,24 +11,53 @@ import (
 	"os/signal"
 	"path"
 	"strings"
+	"sync"
 	"syscall"
 	"time"
 
 	"github.com/ghjm/cmdline"
 	"github.com/google/shlex"
 )
 
+type BaseWorkUnitForWorkUnit interface {
+	CancelContext()
+	Debug(format string, v ...interface{})
```

Comment:
The logger appears to be coupled with the Base work unit here. I wonder if their is a good way to separate them

Reply:
Are you referring to the Debug, Error, Info, and Warning methods?

Reply:
Exactly :smile: